pkg/archive/archive.go

```diff
@@ -660,11 +660,13 @@ func createTarFile(path, extractDir string, hdr *tar.Header, reader io.Reader, L
 	var errors []string
 	for key, value := range hdr.Xattrs {
 		if err := system.Lsetxattr(path, key, []byte(value), 0); err != nil {
-			if err == syscall.ENOTSUP {
+			if err == syscall.ENOTSUP || err == syscall.EPERM {
```

Comment:
`if err == syscall.ENOTSUP || (rsystem.RunningInUserNS() && err == syscall.EPERM)`  ?

Reply:
@cyphar @giuseppe thought?

(either way, the current libcontainer implementation is wrong, because it is also possible to create userns with the full 4G map)

Reply:
Off-topic: is it possible to run any ChromeOS VM with the container feature enabled on macOS?

I tried https://developer.android.com/topic/arc/emulator but I could't enable the "Linux" feature, because it didn't allow me to upgrade the OS to the "Beta"

Reply:
I'd look into running CloudReady in a VM, [they added experimental Linux support recently](https://neverware.zendesk.com/hc/en-us/articles/360001737547-Experimental-Crostini-Support-64bit-Home-Edition-?mobile_site=true).

Linux is actually now available on the stable channel (ChromeOS v69). If you can update your Pixel image to v69, you should be able to enable the Linux (beta) *feature*  in your Settings (yes, it's confusing).

Reply:
thanks, it looks like Android SDK only supports ChromeOS v67. will try cloudready.